lib/create-organization-api.js

```diff
@@ -116,6 +117,56 @@ export default function createOrganizationApi ({ http }) {
       .then((response) => wrapSpaceMembershipCollection(http, response.data), errorHandler)
   }
 
+  /**
+   * Gets an Invitation in Organization
+   * @memberof ContentfulOrganizationAPI
+   * @param {string} id - Invitation ID
+   * @return {Promise<Invitation>} Promise for a Invitation in an organization
+   * @example
+   * const contentful = require('contentful-management')
+   *
+   * client.getOrganizations()
+   * .then((organization) => organization.getInvitation('invitation_id'))
+   * .then((invitation) => console.log(invitation))
+   * .catch(console.error)
+   */
+  function getInvitation (id) {
+    return http.get('invitations/' + id, {
+      headers
+    })
+      .then(response => wrapInvitation(http, response.data), errorHandler)
+  }
+
+  /**
+   * Create an Invitation in Organization
+   * @memberof ContentfulOrganizationAPI
+   * @param {string} id - Invitation ID
+   * @return {Promise<Invitation>} Promise for a Invitation in an organization
+   * @example
+   * const contentful = require('contentful-management')
+   *
+   * const client = contentful.createClient({
+   *   accessToken: '<content_management_api_key>'
+   * })
+   *
+   * client.getOrganizations()
+   *  .then((organization) => organization.createInvitation({
+   *    'email': 'user.email@example.com'
```

Comment:
I think the keys do not need to be quoted here :)

Reply:
@phoebeschmidt yeah, totally, updated, thx :)